c2corg_api/views/waypoint.py

```diff
@@ -28,7 +29,11 @@
 from functools import lru_cache
 from sqlalchemy.orm import joinedload, load_only
 from sqlalchemy.orm.util import aliased
-from sqlalchemy.sql.expression import and_
+from sqlalchemy.sql.elements import literal_column
+from sqlalchemy.sql.expression import and_, union
+
+# the number of routes that are included for waypoints
+NUM_ROUTES = 30
```

Comment:
The idea would be to do it like for the recent outings, show only a small number and give a link to see all (paginated).

Showing 100 routes in the card layout isn't very practical.